ruby/jwt/security/jwt-exposed-credentials-copy.yaml

```diff
@@ -0,0 +1,34 @@
+rules:
+- id: ruby-jwt-exposed-credentials
+  languages:
+  - ruby
+  metadata:
+    cwe:
+    - 'CWE-522: Insufficiently Protected Credentials'
+    owasp:
+    - A02:2017 - Broken Authentication
+    - A04:2021 - Insecure Design
+    source-rule-url: https://r2c.dev/blog/2020/hardcoded-secrets-unverified-tokens-and-other-common-jwt-mistakes/
+    references:
+    - https://cwe.mitre.org/data/definitions/522.html
+    category: security
+    technology:
+    - jwt
+    cwe2021-top25: true
+    subcategory:
+    - audit
+    likelihood: LOW
+    impact: LOW
+    confidence: LOW
+  message: >-
+    Password is exposed through JWT token payload. This is not encrypted and
+    the password could be compromised. Do not store passwords in JWT tokens.
```

Comment:
Heya, I'm rule rascal. I'm an AI so I'm not that good at writing rule messages. But I try. Here's what I think the message could be:

```suggestion
    Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
    Lorem ipsum dolor sit amet.
    Lorem ipsum dolor sit amet.
    Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet.
    Lorem ipsum dolor sit amet.
```